### PR TITLE
serializing additon

### PR DESCRIPTION
**Pull request by manthankumbhar**
### In this PR:
- Have created serializing function and added elements to return to the client-side
- Removed some indentation errors

### CODE REVIEW

**Comment by kavadithya** [comment]
what does "additon" mean?

File: models/Document.py [hunk 1]
```diff
@@ -12,10 +12,13 @@ class Document(db.Model):
...
\ No newline at end of file
+    def __init__(self, id, user_id, data, name, created_at, updated_at):
+        self.id = id
+        self.user_id = user_id 
+        self.data = data
+        self.name = name
+        self.created_at = created_at
+        self.updated_at = updated_at 
+    
+    def __serialize__(self):
```
Comment (kavadithya) [inline models/Document.py:16]: this would not send the right values for datetime objects

File: documents/resources.py [hunk 2]
```diff
@@ -44,9 +44,9 @@ def update_documents(document_id, payload):
...
-        if user_id_from_authorization_header != user_id_from_document_id:
-            return jsonify({'message':'unauthorized request.'}), 400
-        return jsonify(repr(document_row)), 200
\ No newline at end of file
+    user_id_from_authorization_header = payload['user_id']
+    document_row = state_machine.get_document_by_id(document_id)
+    user_id_from_document_id = str(document_row.user_id)
+    if user_id_from_authorization_header != user_id_from_document_id:
+        return jsonify({'message':'unauthorized request.'}), 400
+    return jsonify(document_row.__serialize__()), 200
```
Comment (kavadithya) [inline documents/resources.py:52]: should be doing this for all the API return values

File: models/Document.py [hunk 3]
```diff
@@ -12,10 +12,13 @@ class Document(db.Model):
...
+        return {"id": self.id, "user_id": self.user_id, "data":self.data, "name":self.name, "created_at":self.created_at, "updated_at":self.updated_at}
```
Comment (kavadithya) [inline models/Document.py]: please format / indent the code properly

**Review by kavadithya — COMMENTED** [review]
important comments

**Comment by manthankumbhar** [comment]
@kavadithya
- have added the __serialize__ function inside the class - it's undefined when kept outside, the same thing caused error yesterday!
- have added the function to two places, that's only where the row is sent to the client

File: models/Document.py (hunk 1 above)
Comment (kavadithya) [reply models/Document.py:16]: how is this resolved? @manthankumbhar

**Pull request closed without merging**